docs/source/pages/documentation.md

```diff
@@ -1,351 +1,631 @@
+## Getting Started
 
-Next, the "central_path" argument gives the path to the central machine project. This may be on a local
-filesystem (e.g. if a HPC or network drive is mounted) or using SSH. On Linux systems, ~ syntax is
-not supported and the full filepath must be input.
+Datashuttle provides a Python API and cross-platform command line interface (CLI). In this guide examples will be down using the command line, but corresponding methods can be found in the [API Reference](https://datashuttle.neuroinformatics.dev/pages/api_index.html).
 
-Also required is the "connection_method". This can either be "local_filesystem" or "ssh". If "ssh",
-then the "central_host_id" and "central_host_username" options are also required (see example below).
+The first thing to do when using DataShuttle is to setup a new project on a *local* machine.
 
-The options "use_ephys", "use_behav"... are used to set the data types used on the local PC.
-If these are not set to True, it will not be possible to make data_type folders of this type.
-This option is useful if there are dedicated machines for collection of different data types.
+#### *local* machines and the *central* machine
 
-Finally, the settings "overwrite_old_files_on_transfer", "transfer_verbosity" and "show_transfer_progress" determine
-the behaviour during file transfer. Please see the Data Transfer section for more information.
+DataShuttle makes the distinction between (possibly multiple) *local* machines and a single *central* machine. DataShuttle needs to be setup once for each *local* machine, but requires no setup on the *central* machine.
 
-An example call may look like:
+A typical use case is an experiment in which behavioural data and electrophysiological data are collected on acquisition PCs. They send the data to a central server where it is stored.
 
-```
-project.make_config_file(
-local_path="/path/to/my/my_project",
-central_path="/nfs/nhome/live/username/",
-connection_method="ssh",
-central_host_id="ssh.swc.ucl.ac.uk",
-central_host_username="username",
-overwrite_old_files_on_transfer=True,
-transfer_verbosity="v",
-show_transfer_progress=False,
-use_ephys=True,
-use_behav=True,
-use_histology=True
-)
-```
-or equivalently using the command-line interface
+Later, a subset of the data is transferred to a third machine for analysis. In this case, the behavioural and electrophysiological acquisition machine and analysis machines are 'local'. The central storage machine is the *central* machine.
+
+### One-time project setup on a *local* machine
+
+A one-time setup on each *local* machine used is required, specifying the *project name* and *configurations*.
+
+The configurations tell DataShuttle
+
+- The paths to the *local* and *central* folders that contain the project.
+- How to connect to the central project.
+- The settings that specify how data is transferred.
+- The *data-types* (e.g. *behaviour* (`behav`), *electrophysiology* (`ephys`)) that will be used in the project.
```

Comment:
To avoid double parentheses:
```suggestion
- The *data-types* that will be used in the project, e.g. *behaviour* (`behav`) or *electrophysiology* (`ephys`)
```